subscribie/blueprints/admin/__init__.py

```diff
@@ -1349,6 +1353,41 @@ def set_reply_to_email():
     )
 
 
+@admin.route("/rename-shop", methods=["GET", "POST"])
+@login_required
+def rename_shop():
+    PATH_TO_RENAME_SCRIPT = os.getenv("PATH_TO_RENAME_SCRIPT")
+    PATH_TO_SITES = os.getenv("PATH_TO_SITES")
+    SUBSCRIBIE_DOMAIN = os.getenv("SUBSCRIBIE_DOMAIN")
+    SERVER_NAME = os.getenv("SERVER_NAME")
+    if request.method == "GET":
+        return render_template(
+            "admin/settings/rename_shop.html", SERVER_NAME=SERVER_NAME
+        )
+    elif request.method == "POST":
+        new_name = (
+            request.form.get("new_name", None)
+            .replace(SUBSCRIBIE_DOMAIN, "")
+            .replace(".", "")
+        )
+        NEW_DOMAIN = new_name + "." + SUBSCRIBIE_DOMAIN
+        if new_name is None:
+            return "You must provide a new valid name"
+        p = Path(os.getenv("PATH_TO_SITES") + f"{NEW_DOMAIN}").is_dir()
```

Comment:
This part will look for the files in the directory with the attribute.is_dir() is not adding or editing anything inside.